flash.sh

```diff
@@ -0,0 +1,298 @@
+ADB=${ADB:-adb}
```

Comment:
We use it to flash Gecko and Gaia trough ADB, it's not used to make a full flash but used everytime we want to flash only Gaia and Gecko for example connected to Ethernet.